bin/exec.go

```diff
@@ -77,7 +71,7 @@ func main() {
 
 		// isFound will return the status of experiment in that namespace
 		// 1 -> found, 0 -> not-found
-		isFound := !utils.CheckExperimentInAppNamespace(engineDetails.AppNamespace, engineDetails.Experiments[i], config)
+		isFound := !utils.CheckExperimentInAppNamespace(engineDetails, engineDetails.Experiments[i])
```

Comment:
sorry if it seems like a dumb qn: we seem to pass some attributes directly (engineDetails.Experiments)  as well as passing the whole struct?